packages/web/components/SuccessPlayer.tsx

```diff
@@ -0,0 +1,22 @@
+import { Image, MetaButton, MetaHeading, Text } from '@metafam/ds';
+import React from 'react';
+
+import AvatarImage from '../public/images/avatar.png';
+import { FlexContainer } from './Container';
+import { MetaLink } from './Link';
+
+export const SuccessPlayer: React.FC = () => {
+  return (
+    <FlexContainer h="100%">
+      <MetaHeading m={5}>Success!</MetaHeading>
+      <Image m={10} src={AvatarImage} />
+      <MetaButton mt={5} mb={8}>
+        Set up your profile
+      </MetaButton>
+      <Text fontFamily="heading" color="offwhite">
```

Comment:
I'm using `<Text />` here instead of `<Heading />` since the default fontSize of `<Heading />` is too large, and I think this is cleaner than reducing the fontSize of `<Heading />`.

Reply:
@krzysu @hammadj Do you think we should switch the names of fontFamily of `heading` and `mono`? 
Currently its like so
```
fonts: {
    body: '"IBM Plex Sans", sans-serif',
    heading: '"IBM Plex Mono", monospace',
    mono: '"Press Start 2P"',
  }
```
I suggest we change it to
```
fonts: {
    body: '"IBM Plex Sans", sans-serif',
    mono: '"IBM Plex Mono", monospace',
    heading: '"Press Start 2P"',
  }
```

Reply:
@dan13ram You can customize font size with the Heading component: https://next.chakra-ui.com/docs/components/heading#changing-font-size

Also, I think we want to keep Press Start 2P as the heading font

Reply:
I also find it confusing to have 'mono' set to Press Start and 'heading' set to IBM Plex Mono so I'm for the suggested change, one more thing, let's set fallback for Press Start to sans-serif too, so

```
fonts: {
    body: '"IBM Plex Sans", sans-serif',
    mono: '"IBM Plex Mono", monospace',
    heading: '"Press Start 2P", sans-serif',
  }
```

Reply:
Great. So I will change `Press Start 2P` to be the heading font.